tests/modules/nf-core/checkv/downloaddatabase/test.yml

```diff
@@ -0,0 +1,10 @@
+- name: "checkv downloaddatabase"
+  command: nextflow run ./tests/modules/nf-core/checkv/downloaddatabase -entry test_checkv_downloaddatabase -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/checkv/downloaddatabase/nextflow.config
+  tags:
+    - "checkv"
+    - "checkv/downloaddatabase"
+  files:
+    - path: "output/checkv/test.bam"
+      md5sum: e667c7caad0bc4b7ac383fd023c654fc
+    - path: "output/checkv/versions.yml"
+      md5sum: a01fe51bc4c6a3a6226fbf77b2c7cf3b
```

Comment:
Are you still having problems wit h`create_test_yml`?

Reply:
I was able to ssh into a local desktop (that I can play with) and have been testing it on there. The downloading did also work in there, but some of the additional test workflows (which will now be a seperate module) were not resolved completly yet.